core/src/listener/vrf-L2-request.ts

```diff
@@ -0,0 +1,88 @@
+import { L2Endpoint__factory } from '@bisonai/orakl-contracts'
+import { ethers } from 'ethers'
+import { Logger } from 'pino'
+import type { RedisClientType } from 'redis'
+import { getVrfConfig } from '../api'
+import {
+  L1_ENDPOINT,
+  L2_CHAIN,
```

Comment:
I think we could just reuse `CHAIN`. Am I missing something? 🤔

Reply:
It's just seems more straightforward for me to follow that path. But I understand we should not make some changes in commonly used classes. Let me refactor it the least impactful way to the previous code.

Reply:
Great! Thank you!